WordGame/App/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 25/07/22.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
It seems like there is nothing special going on in `AppDelegate` class.
So you can keep it simple like 
``` swift
@main
class AppDelegate: UIResponder, UIApplicationDelegate {}
```
Later on when you think of adding some functionality here, you can 
write it.